temba/flows/views.py

```diff
@@ -578,6 +620,15 @@ def __init__(self, user, *args, **kwargs):
                     )
 
                     self.fields[Flow.CONTACT_CREATION] = contact_creation
+                elif self.instance.flow_type == Flow.USSD:
+                    ussd_trigger = Trigger.objects.filter(
+                        org=self.instance.org, flow=self.instance, is_archived=False, groups=None,
+                        trigger_type=Trigger.TYPE_USSD_PULL
+                    ).first()
+                    if ussd_trigger:
+                        self.fields['ussd_trigger'].initial = ussd_trigger.keyword
+                        self.fields['ussd_trigger'].widget.attrs['readonly'] = True
+                        self.fields['ussd_trigger'].help_text = 'To change the USSD trigger code, go to Triggers'
```

Comment:
The idea here is that the ussd trigger input here is simply adding a trigger (if you haven't added one yet) or just show it. Updating, deleting would be difficult (eg: cases when you have more triggers for the same flow, etc.), so I decided to keep it simple here and if you want to do updates or deletes you should go to the Trigger section and do it there.